assets/src/js/search.js

```diff
@@ -1,136 +1,159 @@
+  const applyFiltersButton = document.querySelector('.applybtn');
+  applyFiltersButton.onclick = () => {
+    addSelectedFiltersToForm(true);
+    searchForm.submit();
+  };
+
   // Clear single selected filter.
-  $( '.activefilter-tag' ).off( 'click' ).on( 'click', function() {
-    $( '.p4-custom-control-input[value=' + $( this ).data( 'id' ) + ']' ).prop('checked', false );
-    $search_form.submit();
+  const activeFilterTags = [...document.querySelectorAll('.activefilter-tag')];
+  activeFilterTags.forEach(activeFilterTag => {
+    const filterId = activeFilterTag.dataset.id;
+
+    activeFilterTag.onclick = () => {
+      addSelectedFiltersToForm(false, filterId);
+      searchForm.submit();
+    };
   });
 
   // Clear all selected filters.
-  $( '.clearall' ).off( 'click' ).on( 'click', () => {
-    $( 'input[name^="f["]' ).prop( 'checked', false );
-    $search_form.submit();
-  });
+  const clearAllButton = document.querySelector('.clearall');
+  if (clearAllButton) {
+    clearAllButton.onclick = () => {
+      const selectedFilters = [...document.querySelectorAll('input[name^="f["]:checked')];
+      selectedFilters.forEach(selectedFilter => {
+        selectedFilter.checked = false;
+      });
+      searchForm.submit();
+    };
+  }
 
   // Add click event for load more button in blocks.
-  $load_more_button.off( 'click' ).on( 'click', function() {
-    const total_posts    = $(this).data('total_posts');
-    const posts_per_load = $(this).data('posts_per_load');
-    const next_page      = $(this).data( 'current_page' ) + 1;
-    $(this).data( 'current_page', next_page );
-
-    $.ajax({
-      url: localizations.ajaxurl,
-      type: 'GET',
-      data: {
-        action:          'get_paged_posts',
-        'search_query':  $( '#search_input' ).val().trim(),
-        'paged':         next_page,
-        'orderby': $( '#orderby', $search_form ).val(),
-        'query-string':  decodeURIComponent( location.search ).substr( 1 ) // Ignore the ? in the search url (first char).
-      },
-      dataType: 'html'
-    }).done(( response ) => {
-      // Append the response at the bottom of the results and then show it.
-      $( '.multiple-search-result .list-unstyled' ).append( response );
-      $( '.row-hidden:last' ).removeClass( 'row-hidden' ).show( 'fast' );
-
-      if (posts_per_load * next_page > total_posts) {
-        $load_more_button.hide();
-      }
-    }).fail(( jqXHR, textStatus, errorThrown ) => {
-      console.log(errorThrown); //eslint-disable-line no-console
-    });
-  });
+  const navSearchInput = document.getElementById('search_input');
+  const loadMoreButton = document.querySelector('.btn-load-more-click-scroll');
+  if (loadMoreButton) {
+    loadMoreButton.onclick = () => {
+      const { total_posts, posts_per_load, current_page } = loadMoreButton.dataset;
+      const nextPage = parseInt(current_page) + 1;
+      loadMoreButton.dataset.current_page = nextPage;
+
+      const url = localizations.ajaxurl +
```

Comment:
There's an issue with parameters not urlencoded, if you search for `ocean & stuff` for example, it will break the request (based on the `&` as a separator)
![Screenshot from 2022-01-12 11-04-44](https://user-images.githubusercontent.com/617346/149119837-f2972514-e5e7-42f2-877c-8d9697064fa5.png)


It might be safer/simpler to use [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) (through the [URL API](https://developer.mozilla.org/en-US/docs/Web/API/URL_API)) as it encodes url parameters natively, so something like
```js

      let url = new URL(localizations.ajaxurl);
      url.searchParams.append('action', 'get_paged_posts');
      url.searchParams.append('search_query', navSearchInput.value.trim());
      url.searchParams.append('paged', nextPage);
      url.searchParams.append('orderby', orderInput.value);
      url.searchParams.append('query-string', decodeURIComponent(location.search).substring(1));
```
will create a valid url
![Screenshot from 2022-01-12 11-05-31](https://user-images.githubusercontent.com/617346/149120053-f4d25f8d-714b-4381-8e3b-5b11cb9011e0.png)

Reply:
Good catch, thanks, PR updated! 🙂